src/material/form-field/_mdc-text-field-structure-overrides.scss

```diff
@@ -20,6 +20,36 @@
     border: none;
   }
 
+  .mat-mdc-form-field-input-control {
```

Comment:
Are we overriding MDC styles here, or overriding native styles? If so, this file/mixin is only for MDC overrides

Reply:
I thought we already had something, but couldn't find it. Maybe another file? like `user-agent-overrides`? (whatever you prefer here)